docs/shared/guides/nx-and-ts.md

```diff
@@ -0,0 +1,151 @@
+# Nx and TypeScript
+
+> The build system for TypeScript that TypeScript deserves
+
+While you can use Nx with plain JavaScript (using the `--js` flag), Nx has always had first class support for TypeScript. It is the default when you scaffold a new workspace or generate new projects using the native Nx plugins.
+
+To provide even better TypeScript support we introduced the `@nrwl/js` with the intention of providing the **best possible developer experience for the TypeScript ecosystem**.
```

Comment:
It reads more like a blog post rather than a guide. nrwl/js provides the best DX to TypeScript ecosystem

Reply:
Yep indeed. Rephrased it